source/mainnet/net/mobile-wallet/change-passcode-mw.rst

```diff
@@ -9,23 +10,42 @@ If you want to change your passcode or enable/disable your biometrics, you can d
 .. Note::
    You can't change your passcode, password, or biometrics if you have pending identities or accounts in your wallet.
 
-Change your passcode
-====================
+.. tabs::
 
-#. Go to the |morepage| page.
+   .. tab:: |mw-gen2|
 
-#. Select **Update Passcode & Biometrics**.
+      #. Go to the |morepage| page.
 
-   .. image:: ../images/mobile-wallet/MW58.png
-      :width: 25%
+      #. Select **Update Passcode & Biometrics**.
 
-#. Tap **Continue**.
+         .. image:: ../images/mobile-wallet/MW58.png
+            :width: 25%
 
-#. Enter your old passcode or biometrics.
+      #. Tap **Continue**.
 
-#. Enter a new six-digit passcode, or choose **Use full password instead** if you’d rather use a full password.
+      #. Enter your old passcode or biometrics.
 
-#. Choose whether or not to enable biometrics.
+      #. Enter a new six-digit passcode, or choose **Use full password instead** if you’d rather use a full password.
+
+      #. Choose whether or not to enable biometrics.
+
+   .. tab:: |mw-gen1|
+
+      #. Go to the |morepage| page.
+
+      #. Select **Update Passcode & Biometrics**.
+
+         .. image:: ../images/mobile-wallet/MW58.png
```

Comment:
I guess that the gen2 wallet might actually need a new screenshot, that doesn't contain "Make backup" and "Restore backup" for example.